cirq/ops/identity.py

```diff
@@ -147,98 +147,45 @@ def _json_dict_(self):
             **other,
         }
 
+    def _mul_with_qubits(self, qubits: Tuple['cirq.Qid', ...], other):
+        if isinstance(other, raw_types.Operation):
+            return other
+        if isinstance(other, (complex, float, int)):
+            from cirq.ops.pauli_string import PauliString
+            return PauliString(coefficient=other)
```

Comment:
This is surprising here. I think most users would expect to get a `GlobalPhaseOperation`.

Reply:
Acknowledged.

The use case I have in mind is users writing things like `P(a) * -1 * Q(b)` where `P` and `Q` are pauli operations. In that context a pauli string is expected to result for P=I,X,Y,Z. I think it would be reasonable to return a global phase operation, as long as it was capable of pretending to be a pauli string sort of thing...

Reply:
Did you find that `GlobalPhaseOperation` fails to interoperate with the `PauliString`? I think it should, so if it doesn't maybe add a TODO and file an issue. If it does, let's return `GlobalPhaseOperation` from here.